tests/syntax-tests/source/nix/test.nix

```diff
@@ -0,0 +1,17 @@
+{ nixpkgs ? <nixpkgs>
+, nixpkgs' ? import nixpkgs {}}: with nixpkgs';
+
+# some comment
+stdenv.mkDerivation rec {
+  pname = "test";
+  version = "0.2.3";
+  name = "${pname}-${version}";
+
+  src = ./;
+
+  buildInputs = {
+    gzip
+    bzip2
+    python27
+  };
+}
```

Comment:
GitHub syntax highlighting seems to indicate syntax errors here?

Reply:
@sharkdp hi, I have fixed buildInputs arr